src/Library/demos/Application window/main.blp

```diff
@@ -1,47 +1,39 @@
 using Gtk 4.0;
 using Adw 1;
 
-Adw.ApplicationWindow window {
+Gtk.ApplicationWindow window {
   default-width: "800";
   default-height: "600";
-  content: Box {
-    orientation: vertical;
-
-    Adw.HeaderBar {
-      [title]
-      Label {
-        label: "My Awesome App";
-      }
-
-      [end]
-      MenuButton button_menu {
-        menu-model: menu_app;
-        icon-name: "open-menu-symbolic";
-        primary: true;
-      }
+  title: "My App";
+  titlebar: Gtk.HeaderBar {
```

Comment:
updating a root property widget such as `title` now also works as expected - try removing it